app/assets/stylesheets/main.misc.scss

```diff
@@ -1,3 +1,43 @@
+figure.image {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+figure.image span.img.aspect-set {
+  display: block;
+  width: 100%;
+  height: 0;
+  position: relative;
+}
+
+figure.image span.img.aspect-set img {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+figure.image img {
```

Comment:
__[stylelint]__

Expected selector "figure.image img" to come before selector "figure.image span.img.aspect-set img" (no-descending-specificity)


<!-- Comment by [SideCI](https://sideci.com) -->